odoo/addons/base/models/res_users.py

```diff
@@ -624,6 +663,97 @@ def _is_superuser(self):
     def get_company_currency_id(self):
         return self.env.user.company_id.currency_id.id
 
+    def _add_missing_default_values(self, vals):
+        # we need to override this function to keep to have the value of groups_id
+        # supersede the value of inverse computed fields at user creation (in models.py)
```

Comment:
Improve the comment to make it understandable.

Reply:
+1